tests/manage/pv_services/test_run_io_multiple_dc_pods.py

```diff
@@ -1,41 +1,24 @@
 import pytest
-import logging
-import random
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
 from ocs_ci.ocs import constants
-
 from ocs_ci.framework.testlib import ManageTest, tier2
-from ocs_ci.ocs.resources import pod
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_rbd_secret, create_pvcs, create_project,
-    create_cephfs_secret, create_cephfs_storageclass,
-    create_dc_pods, create_serviceaccount)
-
-
-logger = logging.getLogger(__name__)
-
-
-@pytest.fixture()
-def init_pvc_size(request):
-    """
-    Initialize the PVC size for PVC creation
-    """
-    class_instance = request.node.cls
-    class_instance.pvc_size_int = getattr(
-        class_instance, 'pvc_size_int', random.randint(1, 10)
-    )
-    class_instance.pvc_size = f'{class_instance.pvc_size_int}Gi'
 
 
 @tier2
-@pytest.mark.usefixtures(
-    create_project.__name__,
-    create_serviceaccount.__name__,
-    init_pvc_size.__name__,
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-1284")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-1285")
+        )
+    ]
 )
-class BaseRunIOMultipleDcPods(ManageTest):
+class TestRunIOMultipleDcPods(ManageTest):
     """
-    Run IO on multiple dc pods in parallel
+    Run IO on multiple dc pods in parallel - RBD
```

Comment:
just RBD?

Reply:
Rbd and ceph fs both will update doc string